azerothcore/azerothcore-wotlk
# fix(DB/Creature): Centipaar Tunneler attack on summon

Centipaar Tunnelers from Water Survey quest now attack the player after collecting samples. 


The issue suggests that maybe is a missing InhabitType but is not, tried adding water and still the same.


## Changes Proposed:
-  Adding a new SMART SCRIPT to Centipaar Tunneler.  Entry: 5459

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9984

## SOURCE:


## Tests Performed:

- 
- 


## How to Test the Changes:


1. Add the smart script row
2. .go xyz -7218 -2913 6.5 1 1
3. .additem 8584
4. Use the item. The mobs should insta attack you.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
